packages/plugin-table/src/TableDomRenderer.ts

```diff
@@ -0,0 +1,58 @@
+import { AbstractRenderer } from '../../core/src/AbstractRenderer';
+import { TableNode } from './TableNode';
+import { DomRenderingEngine } from '../../plugin-dom/src/DomRenderingEngine';
+
+export class TableDomRenderer extends AbstractRenderer<Node[]> {
+    static id = 'dom';
+    engine: DomRenderingEngine;
+    predicate = TableNode;
+
+    /**
+     * Render the TableNode along with its contents (TableRowNodes).
+     */
+    async render(table: TableNode): Promise<Node[]> {
+        const domTable = document.createElement('table');
+        const domHead = document.createElement('thead');
+        const domBody = document.createElement('tbody');
+        const renderedChildren = await this.renderChildren(table);
+        for (const renderedChild of renderedChildren) {
+            for (const domChild of renderedChild) {
+                // `TableRowDomRenderer` returns a tbody or thead with a tr in
+                // it. Append the tr to `domHead` or `domBody` in function of
+                // what the children are. If we didn't receive a thead or a
+                // tbody, then the child was not parsed by `TableRowDomRenderer`
+                // (probably a technical node, like a script or style) and we
+                // need to simply append the child to `domTable`.
+                if (this._isTableSection(domChild)) {
```

Comment:
This function is duplicated in the parser and renderer. I would have defined a `const tableSectionTagNames` or something like that in the Plugin itself and imported it here to use `includes` in the `if` rather than create a functon just for that.